### ISSUE TITLE
Frontend Docker Failure Due to Script Error

### ISSUE THREAD

**Issue by pupusyae** [ticketz-oss/ticketz#4]
Upon attempting to run the frontend Docker container, an error has been encountered, leading to its failure. The error is related to the execution of a script called envtojson.sh. The specific error messages are as follows:

: not found21:17:17 /ticketz-scripts/envtojson.sh: line 2:
2024-03-10 21:17:17 /ticketz-scripts/envtojson.sh: line 7: syntax error: unexpected end of f

**Comment by allgood** [comment]
Some systems are messing with the shell script that transposes docker `.env-frontend` file to the run time configuration file `config.json`

This can be solved cloning the repository with an special command line:

```sh
git -c core.autocrlf=input clone https://github.com/ticketz-oss/ticketz.git 
```
Unfortunately I do not know how to fix this on the repository side. Will check it later

**Comment by pupusyae** [comment]
To resolve this issue, I took the following steps:

1. First, I opened the Docker Manager and selected the "frontend-1" container.
2. Then, I executed the "View files" action to access the file structure inside the container.
3. I navigated to the "ticketz-scripts" directory and opened the "envtojson.sh" file for editing.
4. After editing the file, I selected all the text, copied it, and then pasted it back into the file.
5. Next, I saved the changes I made.
6. Finally, I restarted the "frontend-1" container.
After completing these steps, I was able to run the "frontend-1" container without any issues. Although I didn't fully understand the root cause of the problem, these steps successfully resolved the issue I encountered.
